WalletWasabi/Crypto/Bip322Signature.cs

```diff
@@ -5,33 +5,32 @@ namespace WalletWasabi.Crypto
 {
 	public class Bip322Signature : IBitcoinSerializable
 	{
-		private Script scriptSig = Script.Empty;
-		private WitScript witness = WitScript.Empty; 
+		private Script _scriptSig = Script.Empty;
 
 		public Bip322Signature(Script scriptSig, WitScript witness)
 		{
-			this.scriptSig = scriptSig;
-			this.witness = witness; 
+			_scriptSig = scriptSig;
+			Witness = witness;
 		}
 
 		public Bip322Signature()
 		{
 		}
 
-		public Script ScriptSig => scriptSig;
+		public Script ScriptSig => _scriptSig;
 
-		public WitScript Witness => witness;
+		public WitScript Witness { get; private set; } = WitScript.Empty;
```

Comment:
Use the field or use `init` instead of `set`.

Reply:
Can't use `init` because we have https://github.com/zkSNACKs/WalletWasabi/pull/5312/files#diff-acffe3461af72121424e627043ba7bbd1a1fac929963a4f806d0da9b628dcb3eR33